jetbrains-core/tst/software/aws/toolkits/jetbrains/core/credentials/SsoSessionConfigurationManagerTest.kt

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.credentials
+
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import software.amazon.awssdk.profiles.ProfileProperty.SSO_ACCOUNT_ID
+import software.amazon.awssdk.profiles.ProfileProperty.SSO_REGION
+import software.amazon.awssdk.profiles.ProfileProperty.SSO_ROLE_NAME
+import software.amazon.awssdk.profiles.ProfileProperty.SSO_START_URL
+import software.aws.toolkits.jetbrains.core.credentials.SsoProfileConstants.SSO_SESSION_PROFILE_NAME
+import software.aws.toolkits.jetbrains.core.credentials.profiles.SsoSessionConstants
+import software.aws.toolkits.jetbrains.core.credentials.profiles.SsoSessionConstants.PROFILE_SSO_SESSION_PROPERTY
+import software.aws.toolkits.jetbrains.core.credentials.profiles.SsoSessionConstants.SSO_SESSION_SECTION_NAME
+
+class SsoSessionConfigurationManagerTest {
+    private lateinit var ssoSessionConfigurationManager: SsoSessionConfigurationManager
+    private val idcProfileName = "testProfile-123456789-TestRole"
+    private val ssoSessionProfileName = "testProfile"
+    private val ssoRegion = "us-west-1"
+    private val ssoStartUrl = "https://example.com"
+    private val scopesList = listOf("scope1", "scope2")
+    private val accountId = "123456789"
+    private val roleName = "TestRole"
```

Comment:
nit: normally these are set per testcase

Reply:
Sure, will set those in the test() function